app/retail/templates/about.html

```diff
@@ -16,6 +16,39 @@
 {% endcomment %}
 {% load i18n static %}
 <!DOCTYPE html>
+<style type="text/css">
```

Comment:
@jakerockland Could we move this into it's own css file ? 
Bonus: You could port over the CSS needed only for `about.html` from `base.css` / `gitcoin.css` to the newly created `about.css`

That way linting picks up any errors in the css